python/raydp/torch/estimator.py

```diff
@@ -209,12 +209,15 @@ def setup(self, config):
                 else:
                     lr_scheduler = None
 
+                # A quick work-around for https://github.com/ray-project/ray/issues/14352
+                # As we only support single model for now, this is OK
                 registered = self.register(
-                    models=model, optimizers=optimizer, criterion=loss, schedulers=lr_scheduler)
+                    models=[model], optimizers=optimizer, criterion=loss, schedulers=lr_scheduler)
```

Comment:
should we wrap the optimizer, loss, lr_scheduler as well?

Reply:
No need to wrap loss. I'm not sure if any optimizer implements Iterable, but I'll wrap them anyway.